src/Forms/Shared/Samples/Map/SetInitialMapArea/readme.md

```diff
@@ -10,7 +10,7 @@ Setting the initial viewpoint is useful when a user wishes to first load the map
 
 ## How to use the sample
 
-As application is loading, initial view point is set and map view opens at the given location.
+When the sample loads, note the map is opened at the initial view point that is set to it.
```

Comment:
This is a little awkward, I wonder if 'When the sample loads, note that the mapview opens at the initial viewpoint defined on the map' would be better? 'set to it' doesn't feel very clear, especially when there's an important document-view distinction to be made.

Reply:
I agree, this came from the common design.